x/lockup/spec/06_hooks.md

```diff
@@ -0,0 +1,16 @@
+<!--
+order: 6
+-->
+
+# Hooks
+
+In this section we describe the "hooks" that `lockup` module provide for other modules.
+
+## Tokens Locked
+
+Upon successful coin lock/unlock, other modules might need to do few actions automatically instead of endblocker basis synchronization.
+
+```go
+  onTokenLocked(address sdk.AccAddress, amount sdk.Coins)
```

Comment:
maybe also pass in the lockDuration and unlockTime.  If the consumer of the hook doesn't need those, it can ignore them

Reply:
added, lockID is added as well